src/Features/CSharp/Portable/AddImport/CSharpAddImportFeatureService.cs

```diff
@@ -395,8 +397,9 @@ private string GetUsingDirectiveString(INamespaceOrTypeSymbol namespaceOrTypeSym
 
             var compilation = await document.Project.GetCompilationAsync(cancellationToken).ConfigureAwait(false);
             var service = document.GetLanguageService<IAddImportsService>();
+            var generator = SyntaxGenerator.GetGenerator(document);
             var newRoot = service.AddImport(
-                compilation, root, contextNode, usingDirective, placeSystemNamespaceFirst, cancellationToken);
+                compilation, root, contextNode, usingDirective, generator, placeSystemNamespaceFirst, cancellationToken);
```

Comment:
feels unneessary.  why would the service not be able to reference the specific generator for the language it is for?